### ISSUE TITLE
Add keystroke to accessibility dialog toolbar button label

### ISSUE THREAD

**Issue by scofalik** [ckeditor/ckeditor5#16921]
## 📝 Provide a description of the improvement

The toolbar button for opening the accessibility dialog is missing a keystroke. The keystroke is displayed in the menu bar.

**Comment by oleq** [comment]
We're good.

<img width="368" alt="Screenshot 2024-08-14 at 11 20 57" src="https://github.com/user-attachments/assets/556c2aa3-eb75-44c7-8c11-24be9b8297f5">
<img width="323" alt="image" src="https://github.com/user-attachments/assets/106c7005-9401-4ff3-be60-0037264e1804">
